lib/runtime-core/Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2018"
 [dependencies]
 nix = "0.12.0"
 page_size = "0.4.1"
-wasmparser = "0.30.0"
+wasmparser = "0.29.2"
```

Comment:
Same. Shouldn't we use the latest version?

Reply:
See: https://github.com/wasmerio/wasmer/pull/437#discussion_r287612425
Trying to minimize differences with cranelift/cranelift master, we should update when they do.